apps/condo/access.js

```diff
@@ -16,6 +16,30 @@ const permissions = {
     canManageRoles: () => true,
 }
 
+async function checkOrganizationPermission (userId, organizationId, permission) {
+    if (!userId || !organizationId) return false
+    const employee = await getByCondition('OrganizationEmployee', {
+        organization: { id: organizationId },
+        user: { id: userId },
+    })
+    if (!employee || !employee.role) return false
+    const employeeRole = await getByCondition('OrganizationEmployeeRole', {
+        id: employee.role,
+        organization: { id: organizationId },
+    })
+    if (!employeeRole) return false
+    return employeeRole[permission] || false
+}
+
+async function checkBillingIntegrationAccessRight (userId, integrationId) {
+    if (!userId || !integrationId) return false
+    const integration = await getByCondition('BillingIntegrationAccessRight', {
+        integration: { id: integrationId },
+        user: { id: userId },
+    })
+    return Boolean(integration && integration.id)
```

Comment:
What will happen when integration.id will equals `0`?

Reply:
It looks like an impossible case. We have database-level constraints

Reply:
```suggestion
    return get(integration, 'id') !== undefined
```